integration-libs/s4-service/order/components/s4-service-order-detail-actions/s4-service-order-detail-actions.component.html

```diff
@@ -1,24 +1,23 @@
-<ng-container *ngIf="order$ | async as order">
-  <div
-    class="cx-order-details-actions row mt-3"
-    *ngIf="displayActions$ | async"
-  >
-    <div class="col-12 d-flex justify-content-end">
-      <span class="cx-action-button">
-        <button
+  <ng-container *ngIf="order$ | async as order">
+    <div class="cx-order-details-actions row  mt-3" *ngIf="checkServiceStatus(order)">
```

Comment:
`major:` what's the reason of removing observable from this place? It's not a good practice to invoke methods in the template as it's going to be called each time this component re-renders. Apart from that, I think `displayActios$ | async` should give same result as `checkServiceStatus(order)` because it uses it under the hood.